### DISCUSSION TITLE
AWS and GCP Authentication

### DISCUSSION THREAD

**Discussion by SurhanZahid** [OpenCSPM/opencspm#181] [Q&A] [answered]
Hey.

Can we integrate AWS and GCP into it? 

What the process? 

or do we have to provide the json file to fetch results every time? 

Need your help with the integration process ASAP.

Will be thankful.

Regards

**Comment by joshlarsen** [comment, marked as answer]
@SurhanZahid you will need to automate the inventory collection process. For AWS, see the example using [AWS Recon](https://github.com/darkbitio/aws-recon/tree/main/utils/terraform). For GCP, see the example [CAI exporter](https://github.com/OpenCSPM/opencspm/tree/main/collection/gcp-cai-exporter).
